python/pyrogue/_DataReceiver.py

```diff
@@ -54,7 +54,6 @@ def __init__(self,
 
         self.add(pr.LocalVariable(name='Data',
                                   typeStr=typeStr,
-                                  disp='',
```

Comment:
@slacrherbst Was removing line#57 (disp='',) an intentional change for this PR?

Reply:
"This also fixes a bug found in the display value for the DataReceiver."

The problem was when we dumped the state the numpy array in the DataReceiver was being exported as a blank line which caused the system to choke when loading in the state as a configuration.

Reply:
but I just realized this is not the correct fix.

Reply:
@slacrherbst Do we want the default to be `groups=['NoState','NoServe','NoStream','NoStatus'],`?  

I am not failiar with all the groups stuff in rogue.  But for some reason you added it to the EPICS data receiver for llrf (see  URL below).  Just checking if we need it here as well.
https://github.com/slaclab/darpa-accel-llrf-phase-1p5/blob/main/firmware/python/darpa_accel_llrf_phase_1p5/_DbgProcessor.py#L32-L35

Reply:
I think you are right about NoStream
. NoStatus does not exist.